src/main.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import App from "./App.tsx";
-import "G:/ReadMei/src/scss/style.scss";
```

Comment:
Using relative path so it doesn't matter where you store the project in your computer

Reply:
oh right thank you i forgor